float32[] frontier
string robotname
---
bool[] is_frontier